WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -210,13 +210,23 @@ class NotificationSettingsViewController: UIViewController {
 
     // MARK: - Error Handling
 
-    fileprivate func handleLoadError() {
-        let title = NSLocalizedString("Oops!", comment: "An informal exclaimation meaning `something went wrong`.")
-        let message = NSLocalizedString("There has been a problem while loading your Notification Settings",
-                                            comment: "Displayed after Notification Settings failed to load")
+    fileprivate func handleLoadError(error: NSError?) {
+        let title = NSLocalizedString(
+            "Failed to load settings",
+            comment: "A message indicating that we couldn't load the user's settings."
+        )
         let cancelText = NSLocalizedString("Cancel", comment: "Cancel. Action.")
         let retryText = NSLocalizedString("Try Again", comment: "Try Again. Action")
 
+        var message = NSLocalizedString(
+            "There has been a problem while loading your Notification Settings",
+            comment: "Displayed after Notification Settings failed to load"
+        )
+
+        if let error {
+            message = error.localizedDescription
```

Comment:
Do you want to `DDLogError` this error too?

Reply:
In this case the error makes its way up to the user, so they should be able to report it.

The other case in this PR doesn't seem to have a good way to bubble up to the user, so I figured logging it was the best we could do there.